projects/js-packages/components/components/pricing-table/styles.module.scss

```
@@ -17,13 +17,44 @@
 	}
 }
 
+.header-logo {
+
+	.is-viewport-large & {
+		margin-top: 32px;
+	}
+
+	margin-bottom: 24px;
+}
+
 .card {
 	margin-top: var(--padding);
 
 	.is-viewport-large & {
 		display: contents;
 	}
 
+	// Border styles for all columns (default gray borders)
+	> :first-child {
+		border-style: solid;
+		border-color: var(--jp-gray-10);
+		border-width: 1.5px 1.5px 0;
+		border-top-left-radius: 8px;
+		border-top-right-radius: 8px;
```

Let us use logical properties instead of positional ones

```suggestion
		border-start-start-radius: 8px;
		border-start-end-radius: 8px;
```